sky/cli.py

```diff
@@ -207,15 +207,36 @@ def _interactive_node_cli_command(cli_func):
                               default=False,
                               required=False,
                               help='Skip confirmation prompt.')
+    retry_until_up = click.option('--retry-until-up',
+                                  '-r',
+                                  is_flag=True,
+                                  default=False,
+                                  required=False,
+                                  help='Retry until up.')
+    region_option = click.option('--region',
+                                 default=None,
+                                 type=str,
+                                 required=False,
+                                 help='The region to use.')
```

Comment:
By a similar token:
`The region to use. If specified, overrides the default region....` (what is default region for AWS, GCP, ...)

Reply:
I don't think there's a default region - if `--region` is not specified SkyPilot will try all regions.

Reply:
that's right, ignore this comment